frontend/src/utils/format_column_data.ts

```
@@ -20,255 +20,152 @@ export type Edge = {
-        parent.group_radius +
-        child.group_radius;
-
-      return [x_y, x_y];
-    }
+    const middle = Math.floor(this.max_width / 2);
 
-    if (lefts.includes(idx)) {
-      // left
-      const x = parent.middle[0] + 10; // Match the parent x
+    let current_x = 1;
+    let row_idx = 1;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'row_idx' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)